### PR TITLE
Fixed subtopics of "Import existing code into a project"

### PR DESCRIPTION
**Pull request by andrewbess**
## Purpose of this pull request

This pull request (PR) fixes subtopics of "Import existing code into a project"

## Affected DevDocs pages

- [Prepare your existing Magento Commerce install](https://devdocs.magento.com/cloud/setup/first-time-setup-import-prepare.html)
- [Import Magento Commerce code into a Cloud project](https://devdocs.magento.com/cloud/setup/first-time-setup-import-import.html)

## Links to Magento source code

-  ...

<!--
If you are fixing a GitHub issue, note it using GitHub keyword format (https://help.github.com/en/articles/closing-issues-using-keywords#closing-an-issue-in-a-different-repository) to close the issue when this pull request is merged. Example: `Fixes #1234`

`master` is the default branch. Merged pull requests to `master` go live on the site automatically. Any requested changes to content on the `master` branch must be related to the released codebase. Any content related to future releases goes in the `develop` branch.

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: src/cloud/setup/first-time-setup-import-prepare.md [hunk 1]
```diff
@@ -137,7 +137,7 @@ Before you push code to the {{site.data.var.ece}} Git repository, modify your `c
 
 To edit `composer.json`:
 
-1. If you haven't done so already, log in to your {{site.data.var.ece}} server as the [Magento file system owner]({{ site.baseurl }}/cloud/before/before-workspace-file-sys-owner.html).
+1. Go to the Magento installation directory of the code that you downloaded from github as the [Magento file system owner]({{ site.baseurl }}/cloud/before/before-workspace-file-sys-owner.html).
```
Comment (meker12) [inline src/cloud/setup/first-time-setup-import-prepare.md]:
```suggestion
1. Go to the Magento installation directory that contains the code that you downloaded from GitHub as the [Magento file system owner]({{ site.baseurl }}/cloud/before/before-workspace-file-sys-owner.html).
```

File: src/cloud/setup/first-time-setup-import-import.md [hunk 2]
```diff
@@ -8,6 +8,10 @@ functional_areas:
 
 Use the following instructions to import your {{site.data.var.ee}} code into your current {{ site.data.var.ece }} project code.
```
[the remark this answers is not in the document]
Comment (andrewbess) [reply src/cloud/setup/first-time-setup-import-import.md]: Fixed @meker12 Please check.

File: src/cloud/setup/first-time-setup-import-import.md [hunk 3]
```diff
@@ -8,6 +8,10 @@ functional_areas:
...
 
+{:.bs-callout-info}
+Before you begin the import process, make sure that you have completed the steps to [prepare your existing Magento Commerce system]({{ site.baseurl }}/cloud/setup/first-time-setup-import-prepare.html).
+
+
 {:.bs-callout-warning}
 When you force push code from an existing Git branch to your {{site.data.var.ece}} project, you overwrite the project code in `master` which removes all data, sites, stores, and other development work you have done on the project. If you want to keep any files or directories, copy them to a directory outside of your project.
```
[the remark this answers is not in the document]
Comment (andrewbess) [reply src/cloud/setup/first-time-setup-import-import.md]: Fixed @meker12 Please check.

File: src/cloud/setup/first-time-setup-import-prepare.md [hunk 4]
```diff
@@ -54,9 +54,9 @@ Add these files to your {{site.data.var.ee}} code:
    -  `<Magento Commerce install dir>/.magento/services.yaml`
    -  `<Magento Commerce install dir>/.magento/routes.yaml`
 
-  For example, to create `<Magento Commerce install dir>/.magento.app.yaml` from the 2.1.4 branch:
+  For example, to create `<Magento Commerce install dir>/.magento.app.yaml` from the 2.3.3 branch:
 
-    1.  In the  {{site.data.var.ece}} GitHub, click [**.magento.app.yaml**](https://github.com/magento/magento-cloud/blob/2.1.4/.magento.app.yaml).
+    1.  In the  {{site.data.var.ece}} GitHub, click [**.magento.app.yaml**](https://github.com/magento/magento-cloud/blob/2.3.3/.magento.app.yaml).
```
[the remark this answers is not in the document]
Comment (andrewbess) [reply src/cloud/setup/first-time-setup-import-prepare.md]: Fixed @meker12 Please check.

File: src/cloud/setup/first-time-setup-import-prepare.md [hunk 5]
```diff
@@ -54,9 +54,9 @@ Add these files to your {{site.data.var.ee}} code:
...
     2.  In the upper right, click **Raw**, as the following figure shows.
 
         ![View the raw version of the file]({{ site.baseurl }}/common/images/cloud_cloud-git_raw.png){:width="600px"}
```
[the remark this answers is not in the document]
Comment (andrewbess) [reply src/cloud/setup/first-time-setup-import-prepare.md]: Fixed @meker12 Please check.

File: src/cloud/setup/first-time-setup-import-prepare.md [hunk 6]
```diff
@@ -80,21 +80,21 @@ You must have an authentication key to access the {{site.data.var.ee}} repositor
 {:.procedure}
 To create a new `auth.json` file:
 
-1. Verify if you have an `auth.json` file, located in your Magento root directory. You can also [get a sample `auth.json`]({{ site.mage2bloburl }}/2.2/auth.json.sample).
+1. Verify if you have an `auth.json` file, located in your Magento root directory. You can also [get a sample `auth.json`]({{ site.mage2bloburl }}/2.3/auth.json.sample).
```
[the remark this answers is not in the document]
Comment (andrewbess) [reply src/cloud/setup/first-time-setup-import-prepare.md]: Fixed @meker12 Please check.

File: src/cloud/setup/first-time-setup-import-prepare.md [hunk 7]
```diff
@@ -80,21 +80,21 @@ You must have an authentication key to access the {{site.data.var.ee}} repositor
...
 
 1. Using a text editor, create an `auth.json` file and save it in your Magento root directory.
```
[the remark this answers is not in the document]
Comment (andrewbess) [reply src/cloud/setup/first-time-setup-import-prepare.md]: Fixed @meker12 Please check.

File: src/cloud/setup/first-time-setup-import-prepare.md [hunk 8]
```diff
@@ -80,21 +80,21 @@ You must have an authentication key to access the {{site.data.var.ee}} repositor
...
-    }
+           "repo.magento.com": {
+               "username": "<public-key>",
+               "password": "<private-key>"
+           }
+       }
+   }
     ```
 
 1. Save your changes to `auth.json` file and exit the text editor.
```
[the remark this answers is not in the document]
Comment (andrewbess) [reply src/cloud/setup/first-time-setup-import-prepare.md]: Fixed @meker12 Please check.

File: src/cloud/setup/first-time-setup-import-prepare.md (hunk 1 above)
Comment (andrewbess) [reply src/cloud/setup/first-time-setup-import-prepare.md]: Fixed @meker12 Please check.

File: src/cloud/setup/first-time-setup-import-import.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (jeff-matthews) [reply src/cloud/setup/first-time-setup-import-import.md]: confirmed

File: src/cloud/setup/first-time-setup-import-import.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (jeff-matthews) [reply src/cloud/setup/first-time-setup-import-import.md]: confirmed

File: src/cloud/setup/first-time-setup-import-prepare.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (jeff-matthews) [reply src/cloud/setup/first-time-setup-import-prepare.md]: confirmed

File: src/cloud/setup/first-time-setup-import-prepare.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (jeff-matthews) [reply src/cloud/setup/first-time-setup-import-prepare.md]: confirmed

File: src/cloud/setup/first-time-setup-import-prepare.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (jeff-matthews) [reply src/cloud/setup/first-time-setup-import-prepare.md]: confirmed

File: src/cloud/setup/first-time-setup-import-prepare.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (jeff-matthews) [reply src/cloud/setup/first-time-setup-import-prepare.md]: confrimed

File: src/cloud/setup/first-time-setup-import-prepare.md (hunk 8 above)
[the remark this answers is not in the document]
Comment (jeff-matthews) [reply src/cloud/setup/first-time-setup-import-prepare.md]: confirmed

File: src/cloud/setup/first-time-setup-import-prepare.md (hunk 1 above)
Comment (jeff-matthews) [reply src/cloud/setup/first-time-setup-import-prepare.md]: confirmed

**Pull request merged**